Makefile

```diff
@@ -17,3 +17,5 @@ docs:
 	python setup.py build_sphinx
 docs-upload: docs
 	python setup.py upload_sphinx
+deps:
+	pip install pyyaml dbapi-compliance==1.15.0
```

Comment:
I would prefer to add them into `requirements-test.txt` (it seems it is quite usual way). So we'll able to use it as on Linux as well as on Windows (in `appveyor.yml`) and will not need to remember to update it in two places.

Reply:
Ok, that makes sense.

Reply:
Please, update it in the `appveyor.yml` file too.

Reply:
Done.